### DISCUSSION TITLE
zerotier-cli access

### DISCUSSION THREAD

**Discussion by szynkowskij** [hassio-addons/app-zerotier#74] [Q&A] [answered]
I'm wanting to add my own moon to my network, but since there are no cli tools, I'm unable to perform the orbit command. I'm trying to work around an issue with a NAT and this seems to be my best course of action. 

I'm probably not understanding the under the the table workings of how Addons work in hassio, but is there a way to add in a moon or gain access to the CLI?

**Comment by frenck** [comment, marked as answer]
Nope, in that case, the add-on is not for you.

**Reply by szynkowskij** [reply]
Thanks for the quick response. Appreciate the work on this otherwise.
